core/templates/dev/head/css/oppia.css

```diff
@@ -60,6 +60,12 @@
   100% { opacity: 1; }
 }
 
+@-webkit-viewport{width:device-width}
```

Comment:
The @-rules (esp. vendor prefixed) cannot be combined, similar to the @- keyframes rules above them. Via https://stackoverflow.com/questions/22780495/grouping-css-keyframes-rules

That was something new to me.

Updated formatting to match existing style.